.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Python Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
Either remove the matrix and just test for one python version or also include `3.10`, `3.11` and `3.12`.

Reply:
New versions will be added after further tests